### PR TITLE
4.17 rosa loaded upgrade

### PR DESCRIPTION
**Pull request by svetsa-rh**
For now installing 4.16.13 builds to test upgrade to 4.17 version.
For future, we should implement this https://issues.redhat.com/browse/OCPQE-26127

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
Hitting some blocker bugs. Will retry in a day or two.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
Still blocked by rosa upgrade issues. Reproducing and data collection in progress.

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
@paigerube14 can I get a lgtm on this one.

Passing classic rosa run [https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_rele[…]de-from-4.16-loaded-upgrade417-24nodes/1833956302984843264](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55897/rehearse-55897-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes/1833956302984843264)

Passing HCP run [https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_rele[…]de-from-4.16-loaded-upgrade417-24nodes/1841572396616650752](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/55897/rehearse-55897-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes/1841572396616650752)

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      GC: "false"
+      HOSTED_CP: "true"
+      ITERATION_MULTIPLIER_ENV: "2"
+      LONG_CLUSTER_NAME_ENABLED: "true"
+      MULTI_AZ: "true"
+      NODE_UPGRADE_TIMEOUT: "14400"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: 4.16.13
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml:50]: is there a way for us to set not this specific version when creating the cluster, rather just 4.16? I have seen this sometimes in past jobs that the verison specified is so out of date

Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml:50]: In this case, it's always picking up 4.16.14 which does not have an upgrade path :( 
To by-pass, I had to hard code.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-416to417
+promotion:
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml:18]: we no longer need the promotion part here as it does nothing for our tests

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml:15]: Similarly here, we don't actually use this image so we can fully take out this section

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.17-candidate-x86-loaded-upgrade-from-4.16-loaded-upgrade417-24nodes

**Comment by svetsa-rh** [comment]
@paigerube14 Can I get an lgtm?

**Pull request merged**